.github/workflows/rebase_latest_version.yml

```diff
@@ -0,0 +1,29 @@
+name: Auto rebase the latest version branch with main
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  synchronise_version_branch:
+    name: Synchronize the latest branch
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Get branch for the latest version
+        run: |
+          echo "DATA_LATEST_VERSION=\
+          $(git for-each-ref --format='%(refname:short)' refs/remotes/origin \
+          | grep -Po '(?<=origin/)v\d.\d(.\d)?$' \
+          | sort -V | tail -n 1\
+          )" >> $GITHUB_ENV
+
+      - name: Rebase and push
+        run: |
+          git switch $DATA_LATEST_VERSION
+          git rebase main
+          git push --force-with-lease
```

Comment:
if this fails we likely need a notification in our updates channel wdyt ?

Reply:
I can do that but I don't think this repo has acces to the slack bot token

Reply:
see with @soonum then please

Reply:
The token has been enabled for this repo, so now it should work